doc/introduction/chemistry.rst

```diff
@@ -99,10 +100,11 @@ the `spin multiplicity <https://en.wikipedia.org/wiki/Multiplicity_(chemistry)>`
 
 .. code-block:: python
 
-    geometry = qchem.read_structure('h2o.SDF')
+    symbols, coordinates = qchem.read_structure('h2o.SDF')
```

Comment:
When there was no `h2o.SDF` file locally, got the following here:
```python
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-6-b4c3ff0d45a7> in <module>
      1 symbols, coordinates = qchem.read_structure('h2o.SDF')
----> 2 hf_file = qchem.meanfield(
      3     symbols,
      4     coordinates,
      5     name='water',

~/xanadu/pennylane/qchem/pennylane_qchem/qchem/structure.py in meanfield(symbols, coordinates, name, charge, mult, basis, package, outpath)
    326     ]
    327 
--> 328     molecule = MolecularData(geometry, basis, mult, charge, filename=path_to_file)
    329 
    330     if package == "psi4":

~/anaconda3/lib/python3.8/site-packages/openfermion/chem/molecular_data.py in __init__(self, geometry, basis, multiplicity, charge, description, filename, data_directory)
    362 
    363         # Name molecule and get associated filename
--> 364         self.name = name_molecule(geometry, basis, multiplicity, charge,
    365                                   description)
    366         if filename:

~/anaconda3/lib/python3.8/site-packages/openfermion/chem/molecular_data.py in name_molecule(geometry, basis, multiplicity, charge, description)
    109 
    110         # Name molecule.
--> 111         name = '{}{}'.format(sorted_info[0][0], sorted_info[0][1])
    112         for info in sorted_info[1::]:
    113             name += '-{}{}'.format(info[0], info[1])

IndexError: list index out of range
```
It could be worth raising an error in such cases, just to alert the user.